shell.nix

```diff
@@ -11,6 +11,7 @@ with import <nixpkgs> {};
       pkgconfig sqlite libsodium boehmgc
       docbook5 docbook5_xsl
       autoconf-archive
+      lsof # Used by the garbage collector to determine what's in use
```

Comment:
The duplication between here and release.nix made me uncomfortable but they do seem pretty duplicatey already.